### PR TITLE
fix: miscellaneous fixes on documentations

### PR DESCRIPTION
**Pull request by yuroitaki**

### CODE REVIEW

File: CHANGELOG.md
Comment (yuroitaki) [inline CHANGELOG.md:1]: removing this since it's not being used for now

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
 [apache-badge]: https://img.shields.io/github/license/saltstack/salt
-[actions-badge]: https://github.com/tlsnotary/tlsn/actions/workflows/ci.yml/badge.svg
+[actions-badge]: https://github.com/tlsnotary/tlsn/actions/workflows/ci.yml/badge.svg?branch=dev
```
Comment (yuroitaki) [inline README.md:11]: the branch is needed, else any failed CI in any branch ll cause a failing status on the README.

File: notary/server/src/domain/notary.rs [hunk 2]
```diff
@@ -48,7 +47,6 @@ pub enum ClientType {
 pub struct SessionData {
     pub max_sent_data: Option<usize>,
     pub max_recv_data: Option<usize>,
-    pub created_at: DateTime<Utc>,
```
Comment (yuroitaki) [inline notary/server/src/domain/notary.rs:51]: removing this since its not being used for now

File: tlsn/examples/simple/README.md [hunk 3]
```diff
@@ -85,6 +85,6 @@ Feel free to try these extra challenges:
 
 ### Next steps
 
-Try out the [Discord example](../Discord/README.md) and notarize a Discord conversations.
+Try out the [Discord example](../discord/README.md) and notarize a Discord conversations.
```
Comment (yuroitaki) [inline tlsn/examples/simple/README.md:88]: the uppercase is causing the link to be invalid on github website

File: CHANGELOG.md
Comment (yuroitaki) [inline CHANGELOG.md:1]: removing this since it's not being used for now

**Review by themighty1 — APPROVED** [review]

**Review by heeckhau — APPROVED** [review]
👍

**Pull request merged**